revenuecatui/src/iosMain/kotlin/com/revenuecat/purchases/kmp/ui/revenuecatui/UIKitPaywall.kt

```diff
@@ -28,12 +29,12 @@ internal fun UIKitPaywall(
         modifier = modifier,
         factory = {
             val viewController = if (footer) RCPaywallFooterViewController(
-                offering = options.offering as RCOffering?,
+                offering = options.offering?.toIosOffering() as? RCOffering,
```

Comment:
This I'm still trying to wrap my head around... The `toIosOffering` returns an object of type `cocoapods.PurchasesHybridCommon.RCOffering`, however, this method expects a `objcnames.classes.RCOffering`. However, doing the cast works fine. Need to figure out the difference between both.